docs/book/src/contributing/README.md

```diff
@@ -153,9 +154,11 @@ To log all communication between the server and the client, there are two choice
   ```
 
 * You can log on the client side, by the `rust-analyzer: Toggle LSP Logs` command or enabling `"rust-analyzer.trace.server": "verbose"` workspace setting.
-  These logs are shown in a separate tab in the output and could be used with LSP inspector.
+  These logs are shown in a separate tab named `rust-analyzer Language Server Trace` in the output and could be used with LSP inspector.
```

Comment:
I wonder if this should be called `rust-analyzer LSP Trace`? Would look more visually different.